src/Raven.Server/Documents/Handlers/Debugging/ServerWideDebugInfoPackageHandler.cs

```diff
@@ -181,32 +179,29 @@ private static void WriteExceptionAsZipEntry(Exception e, ZipArchive archive, st
         }
 
         private async Task<Stream> GetDebugInfoFromNodeAsync(
-            HttpClient httpClient, 
             JsonOperationContext jsonOperationContext,
             string url, 
-            IEnumerable<string> databaseNames)
+            IEnumerable<string> databaseNames,
+            string apiKey)
```

Comment:
You can integrate with @talweiss1982 work in `RavenDB-7466` to avoid needing this.